frontend/lib/resolution/Resolver.cpp

```diff
@@ -1917,36 +1917,37 @@ bool Resolver::enter(const NamedDecl* decl) {
   CHPL_ASSERT(scopeStack.size() > 0);
   const Scope* scope = scopeStack.back();
 
-  // All functions can be overloaded, even parenless ones (via return
-  // intent overloading).
-  bool canOverload = decl->isFunction();
+  LookupConfig config = LOOKUP_DECLS          |
+                        LOOKUP_IMPORT_AND_USE |
+                        LOOKUP_SKIP_PRIVATE_VIS;
```

Comment:
Purely stylistic, but could we write this in
```C++
config |= ...;
comfig |= ...;
```
style?

Reply:
Sure ;)